source/modules/user-settings/authorized-substitutes.rst

```diff
@@ -0,0 +1,8 @@
+Authorized substitutes
+======================
+
+Substitutes are user accounts that are authorized to act on behalf of an other user to validate or refuse a ticket or a change.
+
+The user can choose one or several accounts to delegate the validation task. He may optionnaly limit the date range of the delegation by setting one or both date lmimits.
```

Comment:
```suggestion
The user can choose one or more accounts to delegate the validation task to. He may optionnaly limit the date range of the delegation by setting one or both date limits.
```

I think we cannot replace "he" with "they" because it refers to the user at the beginning of the 1st sentence.

Reply:
"They" can refer to a multiple people or it can also be used to refer to a single person.

Without getting too politically correct, in English we tend to default to gender neutral language when it is not known as opposed to French which seems like it defaults to masculine forms of words.

Reply:
OK, Understood.